src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -185,38 +186,38 @@ def post(self, request, project):
                     {'detail': 'Feedback for this event cannot be modified.'}, status=409)
 
             existing_report.update(
-                name=report.name,
-                email=report.email,
-                comments=report.comments,
+                name=report['name'],
+                email=report['email'],
+                comments=report['comments'],
                 date_added=timezone.now(),
                 event_user_id=euser.id if euser else None,
             )
-            report = existing_report
+            report_instance = existing_report
 
         else:
-            if report.group:
-                report.notify()
+            if report_instance.group:
+                report_instance.notify()
 
-        user_feedback_received.send(project=report.project, group=report.group, sender=self)
+        user_feedback_received.send(project=report_instance.project, group=report_instance.group, sender=self)
 
-        return self.respond(serialize(report, request.user, UserReportWithGroupSerializer(
+        return self.respond(serialize(report_instance, request.user, UserReportWithGroupSerializer(
             environment_func=self._get_environment_func(
                 request, project.organization_id)
         )))
 
-    def find_event_user(self, report):
+    def find_event_user(self, report_data):
         try:
             event = Event.objects.get(
-                group_id=report.group_id,
-                event_id=report.event_id,
+                group_id=report_data.get('group_id'),
+                event_id=report_data['event_id'],
             )
         except Event.DoesNotExist:
-            if not report.email:
+            if not report_data.get('email'):
                 return None
             try:
                 return EventUser.objects.filter(
-                    project_id=report.project_id,
-                    email=report.email,
+                    project_id=report_data['project'].id,
```

Comment:
Does the report no longer have a `project_id` attribute?

Reply:
It still does, we just no long have a `UserReport` model instance at this point, we just have the dictionary from `serializer.validated_data`